File: Лаб/Лаб1/лаб 1-1/ReadFile.cpp
#include <iostream>
#include <fstream>
#include <string>

int main() {
    std::ifstream file("src/sortTestCase.txt");
    
    if (file.is_open()) {
        std::string line;
        if (std::getline(file, line)) {
            std::cout << "Unshsan mur: " << line << std::endl;
        }
        file.close();
    } else {
        std::cerr << "Unable to open file" << std::endl;
    }
    
    return 0;
}

File: Лаб/Лаб1/лаб 1-1/ReadFile.java
import java.io.BufferedReader;
import java.io.FileReader;
import java.io.IOException;

public class ReadFile {
  public static void main(String[] args){
    try (BufferedReader reader = new BufferedReader(new FileReader("C:\\Users\\tsolm\\OneDrive\\Documents\\Алгоритмийн шинжилгээ ба зохиомж\\Лаб\\Лаб1\\лаб 1-1\\sortTestCase.txt"))) {
      String line = reader.readLine();
      System.out.println("Unshsan mur: " + line);
  
    } catch (IOException e) {
        e.printStackTrace();
    }
  }
}
